src/config.ts

```diff
@@ -210,6 +211,42 @@ export interface ValidationSchemaPluginConfig extends TypeScriptPluginConfig {
    * ```
    */
   validationSchemaExportType?: ValidationSchemaExportType
+  /**
+   * @description Uses the full path of the enum type as the default value instead of the stringified value.
+   * @default false
+   *
+   * @exampleMarkdown
+   * ```yml
+   * generates:
+   *   path/to/file.ts:
+   *     plugins:
+   *       - typescript
+   *       - graphql-codegen-validation-schema
+   *     config:
+   *       useEnumTypeAsDefault: true
+   * ```
+   */
+  useEnumTypeAsDefaultValue?: boolean
+  /**
+   * @description Uses the full path of the enum type as the default value instead of the stringified value.
```

Comment:
I think there is a "typo" here, this is the same description as the previous configuration knob.

Reply:
@mat813 Could you send me a PR to fix this?